src/check-c-linkage-decls.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import sys, os
+
+os.chdir (os.environ.get ('srcdir', os.path.dirname (__file__)))
+
+HBHEADERS = os.environ.get ('HBHEADERS', '')
```

Comment:
```suggestion
HBHEADERS = os.environ.get ('HBHEADERS')
```

Reply:
Or shorter (also the all cap variables look weird):
```python
headers = os.environ.get ('HBHEADERS', '').split()
if not headers:
    haeders = [x for x in os.listdir ('.') if x.startswith ('hb') and x.endswith ('.h')]
```

Reply:
Done, excellent